product_profile_example/profile.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+# © 2015 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class ProductProfile(models.Model):
+    _inherit = 'product.profile'
+
+    def _get_types(self):
```

Comment:
Override the field using `selection_add=[('product', 'Stockable Product')]` when you remove this method in the other module.